### ISSUE TITLE
download_utils: Function definition missing for downloading from Google drive

### ISSUE THREAD

**Issue by bauerfe** [neuromorphs/tonic#260]
The function `download_url` in the `download_utils` module attempts to call a function `download_file_from_google_drive`, which is not defined, see [here](https://github.com/neuromorphs/tonic/blob/4b89acb39572f00ac3e36569aa298d6995d6cf3f/tonic/download_utils.py#L123C38-L123C38).

**Comment by biphasic** [comment]
thanks for reporting this. Should we bring it back? Not sure it still works though

https://github.com/neuromorphs/tonic/blob/35f6118850abf7e63b32667f519f0b6f65e68450/tonic/download_utils.py#L120C1-L170C1

**Comment by fabrizio-ottati** [comment]
Usually with the drive stuff it is better if the user does it manually like with Prophesee stuff?

**Comment by bauerfe** [comment]
Yeah I see your pont. I just tried the function from torchvision and at least for the dataset I'm looking at it fails without any warning. 

Then maybe it's better instead to produce an error message, saying that data from google drives should be downloaded manually?

**Comment by biphasic** [comment]
yes agreed!

**Comment by fabrizio-ottati** [comment]
There is also a function from torchdata, but since its development is paused, it is better to provide an error message. What do you think?

**Comment by biphasic** [comment]
I think that makes total sense. Let the user download it manually for now

**Comment by biphasic** [comment]
Just tried out https://github.com/wkentaro/gdown/tree/main and it works super well. Should switch to that for Gdrive
